Content.Shared/Chemistry/ReactionPrototype.cs

```diff
@@ -43,7 +46,11 @@ public void LoadFrom(YamlMappingNode mapping)
             serializer.DataField(ref _name, "name", string.Empty);
             serializer.DataField(ref _reactants, "reactants", new Dictionary<string, ReactantPrototype>());
             serializer.DataField(ref _products, "products", new Dictionary<string, ReagentUnit>());
-            serializer.DataField(ref _effects, "effects", new List<IReactionEffect>());
+
+            if (_moduleManager.IsServerModule)
+            {
+                serializer.DataField(ref _effects, "effects", new List<IReactionEffect>());
+            }
```

Comment:
Slap a TODO on it for when we can make this better.

Is resolving the dependency in LoadFrom also bad?

Reply:
I'm not sure what you mean by resolving the dependency, but the problem is that I want ReactionPrototype in shared to allow predictions, but some implementations of IReactionEffect currently can't be moved to shared.

I think could get around it by making "shared" implementations of each server-only IReactionEffect, but I'd like to handle that in a seperate PR.

Reply:
Yeah my main issue was storing the ref but it's nbd.